aim/web/ui_v2/src/utils/d3/drawParallelAxesBrush.ts

```diff
@@ -0,0 +1,146 @@
+import * as d3 from 'd3';
+import { isNil } from 'lodash-es';
+
+import { IGetAxesScale } from 'types/utils/d3/getAxesScale';
+import { LinesDataType } from 'types/utils/d3/drawParallelLines';
+import {
+  IDrawParallelAxesBrushBrushProps,
+  IFilterDataByBrushedScaleProps,
+  DomainsDataType,
+} from 'types/utils/d3/drawParallelAxesBrush';
+
+function drawParallelAxesBrush({
+  brushRef,
+  plotBoxRef,
+  plotNodeRef,
+  dimensions,
+  data,
+  linesRef,
+  attributesRef,
+}: IDrawParallelAxesBrushBrushProps): void {
+  brushRef.current.xScale = attributesRef.current.xScale;
+  brushRef.current.yScale = { ...attributesRef.current.yScale };
```

Comment:
`brushRef.current.xScale = attributesRef.current.xScale;`

`brushRef.current.yScale = { ...attributesRef.current.yScale };`
  
  why you need to clone yScale function and whats the difference with xScale , can you explain it please ?

Reply:
There is a difference between yScale of Highplot and yScale of LineChart. In HighPlot yScale is an array with multiple yScales. So if I did it without spreading it would have the same reference and we would have a problem with the initial yScales.

Reply:
Thanks